backend/src/personnel/ro/personnel.ro.ts

```diff
@@ -119,12 +118,13 @@ export class PersonnelRO {
   })
   supervisorEmail: string;
 
-  @ApiProperty({
-    description: 'Travel preference',
-    required: true,
-    example: TravelPreference.WILLING_TO_TRAVEL_REGION,
-  })
-  travelPreference: TravelPreference;
+  // TODO - confirm if this should be here - I believe this is on the emcr/bcws data? It was throwing an error
+  // @ApiProperty({
```

Comment:
@fw-noel can you let me know if this should be here? It was throwing an error and it looks like we have this set on  the bcws/emcr side?

Reply:
Yes, this needs to be here for now. They had wanted to add people's preference for travel preference. Not sure why it's throwing an error

Reply:
okee adding back now sorry!